src/utils/artifact/generateAragonArtifact.ts

```diff
@@ -50,40 +46,45 @@ function _generateAragonArtifact(
     abi,
     // Additional metadata
     flattenedCode: './code.sol',
-    appName,
-    appId: appName ? getAppId(appName) : ''
+    appName: appEnsName,
+    appId: getAppId(appEnsName)
   }
 }
 
 // Function Overloading logic
 
 /**
  * Returns aragon artifact.json from app data
+ * @param appEnsName "finance.aragonpm.eth"
  * @param arapp
  * @param abi
  * @param functions Parsed contract function info
  */
 export function generateAragonArtifact(
```

Comment:
Just curious, how does this work? This is how you do overloads in typescript?

Reply:
This is how you define function overloading in Typescript. It's done so the user can see personalized docs depending on the parameters passed